common/src/main/resources/application.conf

```diff
@@ -9,3 +9,7 @@ spray.can {
     verbose-error-messages = "on"
   }
 }
+
+spray.can.server {
+  request-timeout = 35s
+}
```

Comment:
@mars You can put `request-timeout` inside the block above. We don't need to repeat `spray.can.server`. The configuration library will flatten the hierarchical configuration key.